### PR TITLE
internal/fixed harvest.bat

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|-----:|-----:|-------:|
| ETA                  |  0.2  |    0 |     0 |      0 |        0 |
| Developing      |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -   |   -     |        0 |
## Problem description
changed harvest.bat

### CODE REVIEW

**Review by MajoBerger — APPROVED** [review]

File: build-scripts/import/harvest.bat [hunk 1]
```diff
@@ -1,9 +1,9 @@
-IF "%ENVFILE%"=="" set ENVFILE=%cd%/envs/.default
+IF "%ENVFILE%"=="" set ENVFILE=%cd%/../run/envs/.default
 
 :: wiki: https://wiki.lyrasis.org/display/DSDOC7x/OAI#OAI-HarvestingfromanotherDSpace
 pushd ..\..
 :: import community with collection
-docker-compose --env-file %ENVFILE% -p dq-d7 -f docker/cli.yml -v build-scripts/import/assets:/assets run --rm dspace-cli structure-builder -f /assets/test_community_collection.xml -o /assets/test_import_output.xml -e test@test.edu
+docker-compose --env-file %ENVFILE% -p dq-d7 -f docker/cli.yml -v %cd%/build-scripts/import/assets:/assets run --rm dspace-cli structure-builder -f /assets/test_community_collection.xml -o /assets/test_import_output.xml -e test@test.edu
```
Comment (MajoBerger) [inline build-scripts/import/harvest.bat:6]: this change is useless

**Review by MajoBerger — CHANGES_REQUESTED** [review]

**Comment by milanmajchrak** [comment]
@MajoBerger could we close this PR?

**Comment by MajoBerger** [comment]
I don't know. I was never using harvesting... We could ignore it possibly.

**Comment by milanmajchrak** [comment]
Harvesting works, I'm closing it.

**Pull request closed without merging**